.vscode/tasks.json

```diff
@@ -90,6 +90,30 @@
             // You must install unison and unison-fsmonitor
             // curl -L -o unison-fsmonitor https://github.com/TentativeConvert/Syndicator/raw/master/unison-binaries/unison-fsmonitor
             "command": "unison src/extensions jupyterlab/labbox_ephys_widgets_jp/src/extensions -ignore \"Name __pycache__\" -repeat watch"
+        },
+        {
```

Comment:
Some new vscode tasks.
jupyter lab dev: starts jupyter lab in development mode -- watches for changes to the extension (note: you will need to reload the jupyterlab browser page)
labextension install: installs the lab extension
labextension dev: watches for code changes and rebuilds the lab extension